### PR TITLE
Add redux

### PR DESCRIPTION
**Pull request by jasonf7**

### CODE REVIEW

File: src/auth.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        email: email,
+        password: password
+      })
+      .then((res) => {
+        const response = res.data;
+
+        if (response.status === 200) {
+          const token = response.token || 'randomtoken';
+          localStorage.setItem('token', token);
+          cb && cb(true);
```
Comment (c9dong) [inline src/auth.js:17]: ???? explain

File: webpack.production.config.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ module.exports = {
 		}),
 		new HtmlWebpackPlugin({
 			template: './src/template.html',
-			title: 'Webpack App'
+			title: 'ShuttleQL Player Dashboard'
```
Comment (clemmy) [inline webpack.production.config.js:63]: :+1:

File: src/routes.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import App from './components/app';
+import Login from './components/Login/login';
+import Home from './components/Home/home';
+import auth from './auth';
+
+function requireAuth(nextState, replace) {
+  if (!auth.getToken()) {
+    replace({
+      pathname: '/login',
+      state: { nextPathname: nextState.location.pathname }
```
Comment (c9dong) [inline src/routes.js]: dont need this line

File: src/template.html [hunk 4]
```diff
@@ -3,6 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title><%= htmlWebpackPlugin.options.title %></title>
+  <!-- <meta name="description" content=<%= htmlWebpackPlugin.options.title %>> -->
```
Comment (c9dong) [inline src/template.html]: ???

File: webpack.production.config.js (hunk 2 above)
Comment (c9dong) [inline webpack.production.config.js:63]: 👍

**Comment by c9dong** [comment]
🚢

**Pull request merged**